grails-app/domain/org/pih/warehouse/invoice/InvoiceItem.groovy

```diff
@@ -162,7 +139,7 @@ class InvoiceItem implements Serializable {
 
     // Total adjustments value if order adjustment based or total item value if shipment item based
     def getTotalAmount() {
-        return totalAdjustments ?: totalItemPrice
+        return orderAdjustment ? unitPrice : totalItemPrice
```

Comment:
Seems like this is redundant now. InvoiceItem's quantity for adjustments should always be 1, hence you could use only the totalItemprice here (and store always the unit price here https://github.com/openboxes/openboxes/pull/2501/files#diff-f341b7100f86cf5e8f25a73606a5b1d3b49dc9238037ba24cc724889f1bc216dR244, without multiplying it with prepayment percentage)